libntirpc.pc.in.cmake

```diff
@@ -1,11 +1,11 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@
```

Comment:
All of these later CMAKE_INSTALL_PREFIX instances should probably be ${prefix}, according to the pkg-config documentation.

Reply:
Well the right hand side is evaluated and replaced by cmake ... so cmake replaces @CMAKE_INSTALL_PREFIX@ with the prefix directory (say, /usr). pkg-config doesn't enter there yet, just what pkg-config eventually wants. 

FTR, here's an example file that looks correct for me (obtained with this patch and one previous commit backported from next, on top of 1.5.3, for a system-wide installation on Gentoo):
    prefix=/usr
    exec_prefix=/usr
    libdir=/usr/lib64
    includedir=/usr/include/ntirpc
    
    Name: libntirpc
    Description: New Transport Independent RPC Library
    Requires:
    Version: 1.5.3
    Libs: -lntirpc
    Cflags: -I/usr/include/ntirpc

Reply:
The way you have it will work, but .pc files are supposed to use the prefix you set for it's locations, so that they can be overridden via pkg-config --define-prefix.